app/src/pages/playground/ModelConfigButton.tsx

```diff
@@ -156,18 +158,75 @@ export function ModelConfigButton(props: ModelConfigButtonProps) {
   );
 }
 
+interface ModelConfigDialogProps extends ModelConfigButtonProps {}
+function ModelConfigDialog(props: ModelConfigDialogProps) {
+  const instance = usePlaygroundContext((state) =>
+    state.instances.find(
+      (instance) => instance.id === props.playgroundInstanceId
+    )
+  );
+
+  if (!instance) {
+    throw new Error(
+      `Playground instance ${props.playgroundInstanceId} not found`
+    );
+  }
+  const setModelConfigForProvider = usePreferencesContext(
+    (state) => state.setModelConfigForProvider
+  );
+
+  const notifySuccess = useNotifySuccess();
+  const onSaveConfig = useCallback(() => {
+    setModelConfigForProvider({
+      provider: instance.model.provider,
+      modelConfig: instance.model,
+    });
+    notifySuccess({
+      title: "Model Configuration Saved",
+      message: `${ModelProviders[instance.model.provider]} model configuration saved`,
+      expireMs: 3000,
+    });
+  }, [instance.model, notifySuccess, setModelConfigForProvider]);
+  return (
+    <Dialog
+      title="Model Configuration"
+      size="M"
+      extra={
+        <TooltipTrigger delay={0} offset={5}>
+          <Button size={"compact"} variant="default" onClick={onSaveConfig}>
+            Save Config
+          </Button>
+          <Tooltip>
+            Save {ModelProviders[instance.model.provider]} configuration to
+            local storage.
+          </Tooltip>
```

Comment:
```suggestion
          <Tooltip>
            Remember configuration for {ModelProviders[instance.model.provider] ?? "this provider"}.
          </Tooltip>
```

How does this sound? I'd probably avoid mentioning local storage

Reply:
yes much better! Thanks!